spdxconv/main.go

```diff
@@ -0,0 +1,132 @@
+package main
```

Comment:
This one's missing a license ;)

Also maybe move this under `tools/`.

Reply:
Yes, wanted to ask the correct place for this.